Bottom: b2f6b4efd6e7e9e1119dff2019baaec2f9ac7b46
Top:    90c49b82a6756b466cb6a7d3b2735f9e656a62a2
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-11-22 14:52:21 +0800

Refresh of tuntap-napi

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index fdf97be..b470156 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1334,6 +1334,7 @@ static ssize_t tun_get_user(struct tun_struct *tun, struct tun_file *tfile,
 
 	local_bh_disable();
 	napi_schedule(&tfile->napi);
+	/* process bh soon */
 	local_bh_enable();
 
 	stats = get_cpu_ptr(tun->pcpu_stats);
